policies/stable-release-updates.md

```diff
@@ -0,0 +1,81 @@
+Stable Release Update Policy
+============================
+
+This policy covers allowed changes on stable release branches.
+
+Definitions
+-----------
+
+A **stable release** is a patch release from an existing supported minor
+release branch.
+
+A **public interface** is any function, structure or macro declared in a public
+header file.
+
+A **bug fix** is a fix of functionality of the libraries, modules,
+applications, or the build system that (all or any items might apply):
+
+- makes the functionality to conform with the existing end-user documentation
+  where such a change does not break existing common usage
+- makes the end-user documentation to conform with the existing behavior of the
+  software
+
+A **bug fix** is also a fix for an unexpected behavior (not explicitly
+documented in one way or another) of the software when such unexpected behavior
+is a security vulnerability.
+
+A **bug fix** is also a fix for an unexpected behavior of the following kinds
+even if such behavior is not a security vulnerability:
+
+- a memory leak
+- a crash
+- a hang/deadlock or a race condition
+- an out of bounds read or write
+- an uninitialized value read
+- a memory use after free
+- C language undefined behavior
+
+A **bug fix** is also a build failure or test failure fix for platforms that
+are noted as supported for the stated release.
+
+A **bug fix** might also be a fix for an unexpected behavior (not explicitly
+documented in one way or another) where the implementation does not conform to
+existing public specifications that we claim to conform to. Where the
+documentation explicitly contradicts the specifications the documented behavior
+is what matters.
+
+A **functionality change** is **not**:
+
+- performance enhancement
+- memory usage reduction
+- replacement implementation of algorithms
+- refactoring
+
+An **end-user documentation** is not:
+- comments in the code (including public header files)
+- internal documentation (doc/internal)
+
+Changes allowed in stable releases
+----------------------------------
+
+No API or ABI breaking changes are allowed in a minor or patch release.
+
+Only bug fixes including fixes for security issues and end-user documentation
+additions are allowed in stable releases.
+
+The addition of new platforms to LTS branches is acceptable so long as the
+required changes consist solely of additions to configuration.
+
+A documentation addition allowed in stable releases is any addition to the
+documentation that documents the existing behavior of the software. I.e., the
+documentation additions in stable releases cannot add new API contract
+constraints, or implicitly create new bugs in the software by documenting
+behavior that is different from the existing behavior of the software.
+
+Comment or whitespace fixes are allowed only as part of a bug fix to avoid
+later merge conflicts.
+
+Patch release commits are obviously allowed (updates to CHANGES.md, NEWS.md,
+version, and copyright updates).
+
```

Comment:
In 1.1.1 series we had, for example, more strict checks for CA certs. Will it be forbidden when this policy is on?

Reply:
These would have to go through an exemption process and a vote.